util.go

```diff
@@ -38,7 +38,7 @@ func encodeQuery(query map[string][]string) string {
 			continue
 		}
 
-		tuple := key + "=" + oDataEncodeVals(vals)
+		tuple := key + "=" + strings.Replace(strings.Join(vals, ","), " ", "%20", -1)
```

Comment:
Page said

> building the query would indeed be the nicest way to do it, but I think the %20 conversion covers the majority
the user of the go-client would have to manually escape strings though

In order to make the change more palatable (read: less breaking), I've added a `Raw` attribute to the `QueryOption` struct which must be explicitly set to `true` to skip the escaping (which is now handled in `QueryOption.Escape` via the new `encode` argument to `QueryOptions.toMap`).